packages/react-ui/src/components/theme-provider.tsx

```diff
@@ -71,3 +71,28 @@ export const useTheme = () => {
 
   return context;
 };
+
+export const useRipple = () => {
```

Comment:
Remove this, It's related to primereact.

Reply:
I need primereact for the ripple, how else am I going to make it ?